File: apps/server/AliasVault.Client/Utilities/MobileLoginErrorCode.cs
//-----------------------------------------------------------------------
// <copyright file="MobileLoginErrorCode.cs" company="aliasvault">
// Copyright (c) aliasvault. All rights reserved.
// Licensed under the AGPLv3 license. See LICENSE.md file in the project root for full license information.
// </copyright>
//-----------------------------------------------------------------------

namespace AliasVault.Client.Utilities;

/// <summary>
/// Error codes for mobile login operations.
/// These codes are used to provide translatable error messages to users.
/// </summary>
public enum MobileLoginErrorCode
{
    /// <summary>
    /// The mobile login request has timed out after 2 minutes.
    /// </summary>
    Timeout,

    /// <summary>
    /// A generic error occurred during mobile login.
    /// </summary>
    Generic,
}

File: apps/browser-extension/src/entrypoints/popup/types/MobileLoginErrorCode.ts
/**
 * Error codes for mobile login operations.
 * These codes are used to provide translatable error messages to users.
 */
export enum MobileLoginErrorCode {
  /**
   * The mobile login request has timed out after 2 minutes.
   */
  TIMEOUT = 'TIMEOUT',

  /**
   * A generic error occurred during mobile login.
   */
  GENERIC = 'GENERIC',
}
